packages/components/src/components/form/SelectBar/SelectBar.tsx

```diff
@@ -22,10 +22,15 @@ import {
     withFrameProps,
 } from '../../../utils/frameProps';
 import { TransientProps } from '../../../utils/transientProps';
+import { Icon, IconName } from '../../Icon/Icon';
+import { Row } from '../../Flex/Flex';
 
 export const allowedSelectBarFrameProps = ['margin', 'width'] as const satisfies FramePropsKeys[];
 type AllowedFrameProps = Pick<FrameProps, (typeof allowedSelectBarFrameProps)[number]>;
 
+export const fillTypes = ['none', 'default'] as const;
```

Comment:
Perhaps we should create a common fill types like for other props, it's used in many components.